### DISCUSSION TITLE
Color arrows by flow

### DISCUSSION THREAD

**Discussion by rgazelot** [FlowmapBlue/FlowmapBlue#63] [General]
Hello, thanks for this amazing layer. Do you have a plan to have different color schemes for origin/destination flows? I saw that some getters exist on the @flowmap/core package but not on @flowmap/layers.

**Comment by ilyabo** [comment]
True, `getFlowColor`  got lost after the rewrite in v8. I'd still want to add it back, but not sure when I find time for it. Would be happy to accept a PR.

**Reply by rgazelot** [reply]
Perfect, we will do our best to try to contribute, thanks a lot for your work
